### PR TITLE
Restore xdebug

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

File: ci-php/Dockerfile [hunk 1]
```diff
@@ -2,12 +2,19 @@ FROM dailygrommet/php-cli-base
 LABEL description="The Grommet PHP for CI"
 LABEL maintainer="aria@thegrommet.com"
 
+RUN apt-get install -y gnupg2
```
Comment (tmannherz) [inline ci-php/Dockerfile]: This needs to be installed on its own?

File: ci-php/Dockerfile [hunk 2]
```diff
@@ -2,12 +2,29 @@ FROM dailygrommet/php-cli-base
...
+    touch /var/log/xdebug.log && \
+    chown www-data:www-data /var/log/xdebug.log
+
+COPY xdebug.ini /usr/local/etc/php/conf.d/
+
+RUN groupadd --gid 3434 ci \
+  && useradd --uid 3434 --gid ci --shell /bin/bash --create-home ci
+
+# Install composer
+RUN php -r "copy('https://raw.githubusercontent.com/composer/getcomposer.org/master/web/installer', 'composer-setup.php');" &&     php composer-setup.php &&     php -r "unlink('composer-setup.php');" &&     mv composer.phar /usr/local/bin/composer
```
Comment (tmannherz) [inline ci-php/Dockerfile]: Composer is installed in the base image, so this shouldn't be necessary: https://github.com/thegrommet/docker-images/blob/master/php-base/Dockerfile.cli#L21

File: ci-php/Dockerfile (hunk 1 above)
Comment (aredridel) [reply ci-php/Dockerfile]: Yes, because the later depends on it.

**Comment by aredridel** [comment]
@tmannherz This is ready for a final review. I had to use this branch to build the latest image, so it's load bearing at this point.

**Pull request merged**